client/admin/map-pools-reducer.js

```diff
@@ -52,12 +51,7 @@ export default keyedReducer(new MapPoolsState(), {
 
     const history = new MapPoolHistory({
       mapPools: new List(payload.pools.map(p => p.id)),
-      byId: new Map(
-        payload.pools.map(p => [
-          p.id,
-          new MapPool({ ...p, maps: new List(p.maps.map(m => new MapRecord(m))) }),
-        ]),
-      ),
+      byId: new Map(payload.pools.map(p => [p.id, new MapPool({ ...p, maps: new List(p.maps) })])),
```

Comment:
Seems like we're gonna wanna move this reducer to immer ASAP as well, I'm not totally sure of the effects of throwing these JS objects into a map directly

Reply:
I'll keep it near the top of my TODO list :d